x-pack/plugins/maps/public/trigger_actions/visualize_geo_field_action.ts

```diff
@@ -43,6 +46,13 @@ export const visualizeGeoFieldAction = createAction<VisualizeFieldContext>({
   execute: async (context) => {
     const { app, path, state } = await getMapsLink(context);
 
+    const usageCollection = getUsageCollection();
+    usageCollection?.reportUiCounter(
+      APP_ID,
+      'visualize_geo_field',
+      context.originatingApp ? `_${context.originatingApp}` : ''
```

Comment:
Sorry for giving you poor feedback, just saw how this is used `${appName}-${type}-${eventName}` and prefixing with '_' does not make sense. How about just `context.originatingApp ? context.originatingApp : 'unknownOriginatingApp'`